### PR TITLE
Makes it clear whether (in)complete golem shells are free or servant

### PR DESCRIPTION
**Pull request by coiax**
So it's clear whether you're becoming a free or an enslaved golem.

### CODE REVIEW

**Comment by MrPerson** [comment]
Artificial as opposed to all-natural gluten free golem shells?

**Comment by ShadowDeath6** [comment]
Organic golems

**Comment by Penguaro** [comment]
Free range vs caged?

**Comment by GunHog** [comment]
:+1: on differentiating them from Free Golems,
:-1: on the name "Artificial". 

They are golems, all of them are artificial! The lore behind them states (as sourced from the RD "The Liberator" statue) that they were golems aboard a station, but the RD that created them gave them only one order: "Yeah, just go do whatever.".

They seem to have decided to take a ship and roam the stars, eventually (crash?) landing on Lavaland. Judging from the amount of dust on the shells, they went into 'hibernation' during the journey, coming to life during the events of SS13.

From this, we can say that all golems are made the same way, the difference with the Free Golems being that their master decided to let them be free. 

**TL;DR:** Please name them something relating to their ownership status, as ALL golems are artificial!

**Comment by Penguaro** [comment]
So yeah oddly enough, Free Range vs Caged...

**Comment by coiax** [comment]
@XDTM please come up with a different name rather than "artificial".

**Comment by XDTM** [comment]
We could call them servant and free, respectively

`"This is a servant golem shell (Iron)"`
`"This is a free golem shell (Iron)"`

**Comment by factoryman942** [comment]
Also surely the free golems would be the artificial ones since they've literally just been printed in an autolathe
